File: bench/llm/tasks/11-leaderboard/starter.py
player1 = [8, 6, 9, 5, 10, 7]
player2 = [7, 8, 6, 9, 8, 8]
p1_total = 0
p2_total = 0
for score in player1:
    p1_total += score
for score in player2:
    p2_total += score
print("Player 1 total: " + str(p1_total))
print("Player 2 total: " + str(p2_total))

File: bench/llm/tasks/11-leaderboard/starter.go
package main

import "fmt"

func main() {
	player1 := []int{8, 6, 9, 5, 10, 7}
	player2 := []int{7, 8, 6, 9, 8, 8}
	p1Total, p2Total := 0, 0
	for _, score := range player1 {
		p1Total += score
	}
	for _, score := range player2 {
		p2Total += score
	}
	fmt.Println("Player 1 total: " + fmt.Sprint(p1Total))
	fmt.Println("Player 2 total: " + fmt.Sprint(p2Total))
}
